tools/flash-shelly.py

```diff
@@ -338,8 +342,8 @@ def flash_firmware(self):
 
 
 def parse_version(vs):
```

Comment:
let's collect all the various versions we've seen in the wild and that this function needs tp be able to parse.
this will help with refactoring in the future.

Reply:
upto now i have only seen `2.7.0-beta1` that you use and `1.9.3-rc3` that stock use for variants. and now `1.9.2_1L` fot some strange reason Shelly1L stock release fw uses.

Reply:
we will not know if one off untill 1.9.3 is released.